ert_shared/feature_toggling.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import logging
+
+
+class _Feature:
+    def __init__(self, default_enabled, python3_only=False, msg=None):
+        self.is_python3_only = python3_only
+        python_major_version = sys.version_info[0]
+        self.is_enabled = default_enabled and not (
+            python_major_version < 3 and python3_only
+        )
+        self.is_togglable = not (python_major_version < 3 and python3_only)
+        self.msg = msg
+
+
+class FeatureToggling:
+    _conf = {
+        "new-storage": _Feature(
+            default_enabled=False,
+            python3_only=True,
+            msg="New storage starts a http server on your computer serving the \
```

Comment:
How about:
```python
msg = (
    "WARNING: The new storage solution is experimental! "
    "In particular it starts a http server on your computer serving the "
    "data from ERT and should therefore not be used on confidential data."
)
```

Reply:
it is logged as a waring, and thus adding warning to the message looks a bit weird. Removing the warning it looks like this when printed:
`2020-05-12 14:01:20,273 - root - WARNING - The new storage solution is experimental! In particular it starts a http server on your computer serving the data from ERT and should therefore not be used on confidential data.`

Reply:
My mistake 👍